src/gui/tray/UserModel.cpp

```diff
@@ -646,11 +651,12 @@ Q_INVOKABLE void UserModel::openCurrentAccountTalk()
     if (_users.isEmpty())
         return;
 
-    QString url = _users[_currentUserId]->server(false) + "/apps/spreed";
-    if (!(url.contains("http://") || url.contains("https://"))) {
-        url = "https://" + _users[_currentUserId]->server(false) + "/apps/spreed";
+    auto talkApp = _users[_currentUserId]->talkApp();
+    if(talkApp){
+        QDesktopServices::openUrl(talkApp->url());
+    } else {
+        qCInfo(lcActivity) << "The Talk app was not found on the server.";
```

Comment:
nitpick: I'd go for warning instead of info. It's not really supposed to happen, if we got in that else something was wrong in the caller.